README.md

```diff
@@ -17,9 +17,11 @@ To point your docker client to the engine in a local provisioned server:
 
     eval $(docker-machine env snappy-jenkins-local)
 
-The same for the cloud environment:
+The cloud environment uses Docker Swarm as the container scheduler, the command it that case is:
```

Comment:
There's an error: the command IT that case is
maybe: the command FOR that case is